.gitignore

```diff
@@ -1 +1,2 @@
-__pycache__
\ No newline at end of file
+__pycache__
+db.sqlite3
```

Comment:
If you add the db.sqlite3 to the .gitignore, then the database will never be updated in the pull requests, e.g., if you add a prescription on your end, then merge this branch, then I pull it down, that prescription will not show up for me. Maybe this is how we want it, maybe not - wanted to start discussion here